src/components/common/button/ArrowButton.jsx

```diff
@@ -0,0 +1,25 @@
+import icons from '@/assets/icons/icons';
+
+/**
+ * 롤링페이퍼 리스트 화살표 버튼
+ */
+const ArrowButton = ({ onClick, position }) => {
+  const isLeft = position === 'left';
+  const icon = isLeft ? icons.ArrowLeftIcon : icons.ArrowRightIcon;
+
+  return (
+    <button
+      type="button"
+      onClick={onClick}
+      className={`absolute top-1/2 z-20 hidden h-12 w-12 -translate-y-1/2 items-center justify-center rounded-full bg-white shadow-lg transition hover:bg-gray-100 md:flex ${isLeft ? '-left-5' : '-right-5'} `}>
+      <img
+        src={icon}
+        alt={isLeft ? '이전' : '다음'}
+        className="aspect-square h-6 w-6 object-contain"
+        draggable={false}
+      />
+    </button>
+  );
+};
+
+export default ArrowButton;
```

Comment:
이렇게 cva와 cn을 사용해서 스타일링 적용해볼 수 있을 것 같습니다!
icon은 svg로 컴포넌트로 불러와서 img 태그 없이 바로 사용하시면 안깨질 것 같아요!
지금 시안과 디자인이 다르게 적용된 부분들이 있어서 그 부분 추가로 적용해주시면 좋을 것 같습니다.
(border, background blur 효과)

left, right 위치는 아래처럼 수정하시면 조절될 것 같습니다!
부모 기준(RollingPaperList)으로 left, right 위치가 정해져서요
지금 마이너스로 위치를 잡아주셔서 멀리 떨어져보였던 것 같습니다!

```suggestion
import { cva } from 'class-variance-authority';
import icons from '@/assets/icons/icons';
import { cn } from '@/utils/style';

const buttonStyles = cva(
  `absolute top-1/2 z-20 hidden h-10 w-10 -translate-y-1/2 items-center justify-center rounded-full bg-white shadow-lg transition hover:bg-gray-100 md:flex`,
  {
    variants: {
      position: {
        left: 'left-2.5',
        right: 'right-2.5',
      },
    },
  }
);

/**
 * 롤링페이퍼 리스트 화살표 버튼
 */
const ArrowButton = ({ onClick, position }) => {
  return (
    <button
      type="button"
      onClick={onClick}
      className={cn(buttonStyles({ position }))}>
      {position === 'left' ? <icons.ArrowLeftIcon /> : <icons.ArrowRightIcon />}
    </button>
  );
};

export default ArrowButton;
```